evm_loader/test_transaction.py

```diff
@@ -88,6 +113,18 @@ def get_keccak_instruction_and_trx_data(self, data_start):
                                                     )
         return keccak_instruction, trx_data, sign
 
+    def get_keccak_instruction_and_trx_data_2(self, data_start):
```

Comment:
Adapt and reuse `get_keccak_instruction_and_trx_data`

Reply:
Fixed